arroyo-console/src/routes/connections/CreateProfile.tsx

```diff
@@ -27,7 +27,7 @@ export const CreateProfile = ({
   const [error, setError] = useState<string | null>(null);
   const [valid, setValid] = useState<boolean | null>(null);
   const [validating, setValidating] = useState<boolean>(false);
-  const [state, setState] = useState<any | null>(null);
+  const state = useRef<any | null>(null);
```

Comment:
fixes a bug introduced in #454 which caused changes to the form to not always propagate to the state that was actually saved